app/talk/inbox-form.cjsx

```diff
@@ -20,9 +20,11 @@ module.exports = React.createClass
   getInitialState: ->
     validationErrors: []
 
-  validations: (message, subject, recipients) ->
-    userErrors = if (not recipients.length) then ['Messages must have a recipient'] else []
-    subjectErrors = getErrors(subject, subjectValidations)
+  validations: (message) ->
+    recipient_ids = ReactDOM.findDOMNode(@).querySelector('[name="userids"]').value
```

Comment:
See #3409 I'm updating `UserSearch` to get rid of these DOM queries. They aren't guaranteed to work with future versions of `react-select`.